tendrl/commons/flows/base_flow.py

```diff
@@ -156,18 +156,19 @@ def extract_atom_details(self, atom_name):
         namespace = atom_name.split('.objects.')[0]
         object_name = atom_name.split('.objects.')[1].split('.atoms.')[0]
         atoms = self.definitions[namespace]['objects'][object_name]['atoms']
-        atom = atoms[atom_name.split('.')[-1]]
+        atom = atoms[atom_name.split('.')[-2]]
```

Comment:
In https://github.com/anmolbabu/performance_monitoring/blob/bc6e351e2f268aaf1c18d770a3fcf367ad33fe2c/tendrl/performance_monitoring/manager/tendrl_definitions_performance_monitoring.py I didn't require this change as I changed my definition file itself to suit this. So in my definition file I changed the atom names under the objects to class names instead of file names because as per the split in above line it turns out to be looking for the last part of the module path which is the class name and I verified the change to be working.please suggest what is the decided/better approach so that we can make changes accordingly